Dockerfile

```diff
@@ -1,11 +1,45 @@
+# Build Smithy CLI using latest JDK
+FROM gradle:7-jdk17 AS build
```

Comment:
Is this image available on https://gallery.ecr.aws/?

Perhaps https://gallery.ecr.aws/bitnami/gradle

Reason because I have seen docker [rate limiting](https://www.docker.com/increase-rate-limits/) IP ranges and while connected to Amazon VPN, it leads to failures.

Reply:
Good to know! It seems to be mirrored here: https://gallery.ecr.aws/docker/library/gradle. I will swap it out 👍